code/processes/rdb.q

```diff
@@ -110,6 +112,8 @@ endofday:{[date]
 	
 reload:{[date]
 	.lg.o[`reload;"reload command has been called remotely"];
+	/-set timeout to be 0 before tables are wiped
+	system"T 0";
```

Comment:
You can't do this here and have it take effect ... but I think this is un-necessary? The timer will set it to 0, and it will then be set back to what it should've been when the reload is complete